docker/entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# default variables
+: "${SLEEP:=1}"
+: "${TRIES:=60}"
+
+function wait_for_database {(
+  echo "Waiting for database to respond..."
+  tries=0
+  while true; do
+    [[ $tries -lt $TRIES ]] || return
+    (echo "from django.db import connection; connection.connect()" | umap shell) >/dev/null
+    [[ $? -eq 0 ]] && return
+    sleep $SLEEP
+    tries=$((tries + 1))
+  done
+)}
+
+# first wait for the database
+wait_for_database
+# then migrate the database
+umap migrate
+# then collect static files
+umap collectstatic --noinput
```

Comment:
nit: if static files have already been collected, is this a no-op?

Reply:
Yes, Django will compare collected files with the source and just display a message if everything is up to date.

```
$ umap collectstatic --no-input
Loaded local config from /home/ybon/Code/py/umap/umap/settings/local.py

114 static files copied to '/var/local/umap/static', 241 unmodified.
$ umap collectstatic --no-input
Loaded local config from /home/ybon/Code/py/umap/umap/settings/local.py

0 static files copied to '/var/local/umap/static', 355 unmodified.
```